City-of-Helsinki/drupal-helfi-sote
# UHF-8512: Add ibmchatapp configs to configignore

# [UHF-8512](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8512)
Add ibmchatapp blocks to config ignore in order to administrate them directly from ui

## What was done
* Add ibmchatapp blocks to config ignore
* Remove the ignored configs

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8512-config-ignore-sotebot`
  * `make fresh`
* Run `make drush-cr`

## How to test
After installing, you should still be seeing the sotebot popup on frontpage.

To do some additional testing, try: 
* Opening the [ibmchatapp block admin page](https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/admin/structure/block/manage/ibmchatapp?destination=/fi/sosiaali-ja-terveyspalvelut/admin/structure/block) and deleting some the pages in the visibility settings. When vewing these pages **you should see the block** as the condition is negated.
* Do the same for the [exceptions block](https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/admin/structure/block/manage/ibmchatapp_watson_sotebot_exceptions?destination=/fi/sosiaali-ja-terveyspalvelut/admin/structure/block). When removing pages from this block, you should stop seeing them on the page,
* Finally, try running `drush cim -y` on your machine. The changes to the blocks you just made should not be overwritten, since the configs are ignored. 

## Designers review





[UHF-8512]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
